main.js

```diff
@@ -50,27 +50,27 @@ const featureData = [
   },
 
   {
-    name: 'Ryan Merkley',
-    image: 'assets/images/speaker_06.png',
-    position: 'CEO of Creativve Commons, ex COO of the Mozilla Foundation',
+    name: 'Lila tretikov',
+    image: 'assets/images/speaker_03.png',
+    position: 'Executive Director of the Wikimedia Foundation',
     archievement:
-      'Ryan had been leading open-source projects at the Mozilla Foundation such as the open-source move-ment',
+      'Lila Tretikov is the Executive of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely available in 290 languag-es and used by nearly half a billion people around the world every month.',
   },
 
   {
-    name: 'Ryan Merkley',
-    image: 'assets/images/speaker_06.png',
-    position: 'CEO of Creativve Commons, ex COO of the Mozilla Foundation',
+    name: 'SohYeong Noh',
+    image: 'assets/images/speaker_02.png',
+    position: 'Director of Art Centre Nabi and a board member of CC Korea',
     archievement:
-      'Ryan had been leading open-source projects at the Mozilla Foundation such as the open-source move-ment',
+      'Executive Director of the Wikimedia Foundation',
```

Comment:
Great design @maths29 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the design given. I would suggest you add an extra margin to the left and right of your speakers' section so that your design can look like the design given.
- Your design
![margin](https://github.com/maths29/Capstone-project-1/assets/63917725/07ef1579-2693-4ba3-a72b-d30df6f9f305)
- Given design
![Given sspeakers](https://github.com/maths29/Capstone-project-1/assets/63917725/d22ad8ab-c5ed-4f32-88a6-e5dae1f52577)